content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -32,7 +32,7 @@ The {{< glossary_tooltip text="CSI" term_id="csi" >}} Volume Cloning feature add
 
 A Clone is defined as a duplicate of an existing Kubernetes Volume that can be consumed as any standard Volume would be.  The only difference is that upon provisioning, rather than creating a "new" empty Volume, the back end device creates an exact duplicate of the specified Volume.
 
-The implementation of cloning, from the perspective of the Kubernetes API simply adds the ability to specify an existing unbound PVC as a dataSource during new pvc creation. 
+The implementation of cloning, from the perspective of the Kubernetes API simply adds the ability to specify an existing bound PVC as a dataSource during new pvc creation.
```

Comment:
should add comma after "API" and before "simply".

Reply:
Done.